src/content/docs/new-relic-solutions/get-started/glossary.mdx

```diff
@@ -66,12 +66,40 @@ Whether you're considering New Relic or you're already using our capabilities, t
     id="agent"
     title="agent"
   >
-    A New Relic agent is an installable piece of software that integrates with multiple types of technologies (for example, web frameworks, operating systems, and types of databases) and reports data to New Relic, usually on a specific cadence. We have <InlinePopover type="apm" /> agents, an infrastructure agent, a <InlinePopover type="browser" /> agent, and mobile monitoring agents. The various technologies that our agents integrate with are just some of [our many integrations](https://newrelic.com/instant-observability).
+    A New Relic agent is an installable piece of software that integrates with multiple types of technologies (for example, web frameworks, operating systems, and types of databases) and reports data to New Relic, usually on a specific cadence. All agents are integrations, but not all integrations are agents. Many of our agents activate other integrations (the infra agent activates the Cassandra integration).
+
+Here's a list of what we consider New Relic agents:
+
+* APM agents:
+	* Java
+	* PHP
+	* Python
+	* Ruby
+	* Go
+	* .Net
+	* Node.js
+	* C 
```

Comment:
I think it's C SDK, but it's more or less deprecated, maybe? I could also be wrong but i think we stylizle it as .NET